src/metabase/api/permission_graph.clj

```diff
@@ -94,8 +107,9 @@
    [:map-of
     Id
     [:map
+     [:view-data {:optional true} Schemas]
+     [:create-queries {:optional true} Schemas]
      [:data {:optional true} "DataPerms"]
```

Comment:
I'm a bit confused about the removal of `:query` here - was that unused?

Reply:
Yeah, `:query` was unused. That was an artifact of our last attempt to do this split, before the refactor